canonical-web-and-design/juju.is
# add redirects to new about page, remove old about pages

## Done

- Redirected all old pages within the /about directory to the new /about page
- Removed associated templates and routes

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Try to visit:
  - https://juju-is-197.demos.haus/architecture
  - https://juju-is-197.demos.haus/beyond-configuration-management
  - https://juju-is-197.demos.haus/devsecops
  - https://juju-is-197.demos.haus/high-availability-enterprise-olm
  - https://juju-is-197.demos.haus/hosted-olm
  - https://juju-is-197.demos.haus/integration
  - https://juju-is-197.demos.haus/mission
  - https://juju-is-197.demos.haus/model-driven-operations
  - https://juju-is-197.demos.haus/multi-cloud-operations
  - https://juju-is-197.demos.haus/operator-lifecycle-manager
  - https://juju-is-197.demos.haus/operator-services
  - https://juju-is-197.demos.haus/ops-code-quality
  - https://juju-is-197.demos.haus/overview
  - https://juju-is-197.demos.haus/universal-operators
 
See that you are redirected to /about in each case

## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/2007
